source/user-manual/agents/remote-upgrading/create-custom-wpk-automatically.rst

```diff
@@ -0,0 +1,102 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _create-custom-wpk-automatically:
+
+Automated custom WPK packages creation
+=======================================
+
+Wazuh provides an automated way of building WPK packages using docker so there is no need for any other dependency.
+
+To create a WPK package this way only docker and a X509 certificate are needed, in the case of Windows WPK a msi package is needed.
+
+1. Get a X509 certificate and CA
+--------------------------------
+
+1. Create root CA
+
+.. code-block:: console
+
+    # openssl req -x509 -new -nodes -newkey rsa:2048 -keyout wpk_root.key -out wpk_root.pem -batch
+
+2. Create a certificate and key
+
+.. code-block:: console
+
+    # openssl req -new -nodes -newkey rsa:2048 -keyout wpkcert.key -out wpkcert.csr -subj '/C=US/ST=CA/O=Wazuh'
+
+3. Set the location as follows:
+
+    - ``/C=US`` is the country.
+    - ``/ST=CA`` is the state.
+    - ``/O=Wazuh`` is the organization's name.
+
+4. Sign this certificate with the root CA
+
+.. code-block:: console
+
+    # openssl x509 -req -days 365 -in wpkcert.csr -CA wpk_root.pem -CAkey wpk_root.key -out wpkcert.pem -CAcreateserial
+
+2. Build WPK
+-------------
+
+Requirements
```

Comment:
This sub-sub section must be removed. You can write it as normal text. In addition, you are saying that you only need Docker, and 3 lines later you are using `git`.

Reply:
Done in d3a58a1, 473937b, ff99d41